### PR TITLE
[Terraform] Add account-level examples for WAF managed rules and rate limiting

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-5804.

### CODE REVIEW

File: content/terraform/additional-configurations/rate-limiting-rules.md [hunk 1]
```diff
@@ -55,6 +55,66 @@ resource "cloudflare_ruleset" "zone_rl" {
...
 ```
 
+<details>
+<summary>Account-level example configuration</summary>
+<div>
+
+{{<Aside type="note" header="Before you start">}}
+* Account-level rate limiting configuration requires an Enterprise plan with a paid add-on.
+
+* Custom rulesets deployed at the account level will only apply to incoming traffic of zones on an Enterprise plan. The expression of your `execute` rule must end with `and cf.zone.plan eq "ENT"`.
```
Comment (RebeccaTamachiro) [inline content/terraform/additional-configurations/rate-limiting-rules.md:65]:
```suggestion
* Custom rulesets deployed at the account level will only apply to incoming traffic of zones on an Enterprise plan. The expression of your `execute` rule must end with `and cf.zone.plan eq \"ENT\"`.
```

File: content/terraform/additional-configurations/waf-managed-rulesets.md [hunk 2]
```diff
@@ -69,6 +69,52 @@ resource "cloudflare_ruleset" "zone_level_managed_waf" {
...
 ```
 
+<details>
+<summary>Account-level example configuration</summary>
+<div>
+
+{{<Aside type="note" header="Before you start">}}
+* Account-level WAF configuration requires an Enterprise plan with a paid add-on.
+
+* Managed rulesets deployed at the account level will only apply to incoming traffic of zones on an Enterprise plan. The expression of your `execute` rule must end with `and cf.zone.plan eq "ENT"`.
```
Comment (RebeccaTamachiro) [inline content/terraform/additional-configurations/waf-managed-rulesets.md:79]:
```suggestion
* Managed rulesets deployed at the account level will only apply to incoming traffic of zones on an Enterprise plan. The expression of your `execute` rule must end with `and cf.zone.plan eq \"ENT\"`.
```

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/terraform/additional-configurations/rate-limiting-rules.md (hunk 1 above)
Comment (pedrosousa) [reply content/terraform/additional-configurations/rate-limiting-rules.md:65]: As discussed outside GitHub, for now let's not escape these sub-expressions in the middle of a sentence, only in full examples.

File: content/terraform/additional-configurations/waf-managed-rulesets.md (hunk 2 above)
Comment (pedrosousa) [reply content/terraform/additional-configurations/waf-managed-rulesets.md:79]: As discussed outside GitHub, for now let's not escape these sub-expressions in the middle of a sentence, only in full examples.

**Pull request merged**